pkg/key/key.go

```diff
@@ -3,7 +3,8 @@ package key
 // These consts are used by template.Params.Validate() to check that provided versions
 // fall within the known-good ranges.
 const (
-	CalicoVersionConstraint                       = ">= 3.15.0 < 3.16.0"
+	CalicoPolicyOnlyVersionConstraint             = ">= 3.15.0 < 3.16.0"
```

Comment:
I don't want to update and test `calico-policy-only.yaml` with Calico 3.19 so I'm splitting the version constraint depending on `CalicoPolicyOnly` parameter passed into the cloudconfig template.